.github/workflows/coordinator-test.yml

```diff
@@ -6,16 +6,25 @@ on:
 jobs:
   integration-test:
     name: Integration Test
-    # CCM installation trickier on 20.04, alas; Python 2.x dep etc problematic
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
       # Defaults to "true" but let's let all runs finish
       # (see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)
       fail-fast: false
       matrix:
         backend: [ cassandra-40, cassandra-311, dse-68 ]
+        include:
+          - name: cassandra-40
+            ccm-version: -v 4.0.7
+          - name: cassandra-311
+            ccm-version: -v 3.11.14
+          - name: dse-68
+            ccm-version: -v 6.8.31 --dse
```

Comment:
dse version should be 6.8.32?

Reply:
I think there's a way to do this without specifying any specific versions in this file. Pushed another commit to give it a try.